### ISSUE TITLE
Feature Request: per check debug option

### ISSUE THREAD

**Issue by douglasawh** [NagiosEnterprises/nagioscore#331]
I believe the subject is likely all you need. Currently if you turn on debug you get a lot of info...maybe which you don't want.

This request is not to suggest there should not be a global debug option.

**Comment by jfrickson** [comment]
If we want to do that, it will require changes to several structures, so it will have to wait for core 5.

**Comment by hedenface** [comment]
I don't think this makes much sense, actually - the whole reason for debug is to debug what is happening. Trace messages up to and after a particular service (or host or object or whatever) are generally going to be just as important as the ones relating specifically to the service (or host or object or whatever).

We'll bring it up at the next meeting regardless and see what the consensus is.
